packages/client/src/comp/states/Arizona.tsx

```diff
@@ -3,42 +3,54 @@ import React from 'react'
 import Select from 'muicss/lib/react/select'
 import Option from 'muicss/lib/react/option'
 
+
 import { ArizonaInfo, isArizonaParty, arizonaParty, arizonaIdentityType, isArizonaIdentity } from '../../common'
 import { Base } from './Base'
 import { useControlRef } from '../util/ControlRef'
 import { BaseInput } from '../util/Input'
 
+const arizonaPrimary = process.env.REACT_APP_ARIZONA_PRIMARY
+
 export const Arizona = () => {
   const partyRef = useControlRef<Select>()
   const idTypeRef = useControlRef<Select>()
   const idDataRef = useControlRef<Select>()
 
   return <Base<ArizonaInfo>
     enrichValues={(info) => {
-      const party  = partyRef.value()
-      if (!isArizonaParty(party)) return null
-
       const idType  = idTypeRef.value()
       if (!isArizonaIdentity(idType)) return null
 
       const idData = idDataRef.value()
       if (!idData) return null
 
+      if (arizonaPrimary) {
```

Comment:
Let's not rebuild the return object (there are too many values here).  Let's have a `base= { ...}` and `return base` or `return {...base, party}`.  Still need
```
if (arizonaPrimary && !isArizonaParty(party)) return null
```